js/core/utils/string.js

```diff
@@ -94,9 +94,14 @@ var isEmpty = (function() {
     };
 })();
 
+var capitalize = function(text) {
```

Comment:
Can we use [inflector.titleize](https://github.com/DevExpress/DevExtreme/blob/3481c3caa6a6e8fe2d5600f447001e8cd0083ddd/js/core/utils/inflector.js#L42-L46) instead?